srcpkgs/python3-imageio/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'python3-imageio'
+pkgname=python3-imageio
+version=2.6.1
+revision=1
+wrksrc=imageio-${version}
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3"
```

Comment:
There is more depends. You can check by installing it into clean masterdir of xbps-src and playing a bit.

Reply:
Oooorrr I could have been smarter first and checked setup.py.  Itchy trigger finger.  Will correct.